src/components/Article.jsx

```diff
@@ -60,6 +60,11 @@ export default function (props) {
         <button id='commentsbutton' onClick={(event) => {
           setShow(prevShow => !prevShow)
         }}><ButtonSwitch show={showComments}/></button>
+        <LeaveComment show={showComments} id={currentId} onSubmit={(event) => {
+          getComments(currentId).then((response) => {
```

Comment:
In this onSubmit, you shouldn't be making an api call, this should be happening in a useEffect. It is more efficiently to only reset the comments in the onSubmit.